ansible/roles/operator-pipeline/templates/openshift/tasks/run-static-tests.yml

```diff
@@ -86,7 +89,8 @@ spec:
           --output-file "$JSON_RESULTS_FILE" \
           --repo-path "$(workspaces.source.path)" \
           --suites "$(params.test_suites)" $EXTRA_ARGS \
-          "$(params.operator_name)" "$(params.bundle_version)"
+          "$(params.operator_name)" "$(params.bundle_version)" \
+          "$(params.affected_catalog_operators)"
```

Comment:
Just making sure here.. Did you test if this also works for non-catalog changes? Will an empty string be handled correctly?

Reply:
I tested non-catalog changes by running integration tests manually [here](https://github.com/redhat-openshift-ecosystem/operator-pipelines-test/pull/2040). The test already work with empty string when `bundle_version` is empty. It also work for catalog changes, so I expect empty strings are handled correctly and passed as positional arguments.